Rock/Model/LocationService.Partial.cs

```diff
@@ -42,29 +41,15 @@ public partial class LocationService
         /// <returns>
         /// The first <see cref="Rock.Model.Location" /> where an address match is found, if no match is found a new <see cref="Rock.Model.Location" /> is created and returned.
         /// </returns>
-        public Location Get( string street1, string street2, string city, string state, string postalCode, string country, bool verifyLocation = true )
+        public Location Get( string street1, string street2, string city, string state, string postalCode, string country, bool verifyLocation = true, Group group = null )
         {
             // Make sure it's not an empty address
             if ( string.IsNullOrWhiteSpace( street1 ) )
             {
                 return null;
             }
 
-            // First check if a location exists with the entered values
-            Location existingLocation = Queryable().FirstOrDefault( t =>
-                ( t.Street1 == street1 || ( ( street1 == null || street1 == "" ) && ( t.Street1 == null || t.Street1 == "" ) ) ) &&
-                ( t.Street2 == street2 || ( ( street2 == null || street2 == "" ) && ( t.Street2 == null || t.Street2 == "" ) ) ) &&
-                ( t.City == city || ( ( city == null || city == "" ) && ( t.City == null || t.City == "" ) ) ) &&
-                ( t.State == state || ( ( state == null || state == "" ) && ( t.State == null || t.State == "" ) ) ) &&
-                ( t.PostalCode == postalCode || ( ( postalCode == null || postalCode == "" ) && ( t.PostalCode == null || t.PostalCode == "" ) ) ) &&
-                ( t.Country == country || ( ( country == null || country == "" ) && ( t.Country == null || t.Country == "" ) ) ) );
-            if ( existingLocation != null )
-            {
-                return existingLocation;
-            }
-
-            // If existing location wasn't found with entered values, try standardizing the values, and
-            // search for an existing value again
+            // Standardize the location for comparison to other locations
             var newLocation = new Location
```

Comment:
We should probably keep this. Would rather take the small db hit to reduce chance address already exists without being standardized.